src/main/java/uk/gov/cshr/service/InviteService.java

```diff
@@ -14,14 +17,15 @@
 import java.util.Date;
 import java.util.HashMap;
 
+@Service
+@Transactional
 public class InviteService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(InviteService.class);
     private static final String EMAIL_PERMISSION = "email";
     private static final String ACTIVATION_URL_PERMISSION = "activationUrl";
     private static final long EXPIRATION_IN_MILLISECONDS = 86400000; // 24hrs = 86400000ms
```

Comment:
CHORE - This should come from configuration

Reply:
Just to clarify, do you just mean the EXPIRATION_IN_MILLISECONDS, or all 3 fields?

Reply:
just EXPIRATION_IN_MILLISECONDS as comment for the line.